tools/dataset/labelbee_to_coco.py

```diff
@@ -0,0 +1,154 @@
+import os
+import json
+import shutil
+import argparse
+
+'''
+    将 labelbee多步标注 后的关键点数据进行标签转化，转至mmpose模型需要的coco格式。 数据集先划分好。
+'''
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--input-file', type=str, default='/home/wanghao/Work/labelbee_json_2_MMPose_COCO_json/', help='输入labelbee json的根路径')
+    parser.add_argument('--output-file', type=str, default='/home/wanghao/Work/labelbee_json_2_MMPose_COCO_json/', help='输出mmpose json文件根路径')
+    parser.add_argument('--dataset-split', type=list, default=['train', 'val'], help='数据集划分，对应labelbee根路径下划分情况')
+    args = parser.parse_args()
+
+
+    data_set = args.dataset_split
+    for n in range(len(data_set)):
+        anno_path = args.input_file + data_set[n] + "/"
+        anno_list  = os.listdir(anno_path)
+
+        sort_anno_list = []
+        for i in range(len(anno_list)):
+            sort_anno_list.append(int(anno_list[i].split(".jpg")[0]))
+        sort_anno_list = sorted(sort_anno_list)
+
+        anno_list = []
+        for i in range(len(sort_anno_list)):
+            anno_list.append(str(sort_anno_list[i]) + ".jpg.json")
+        print(anno_list)
+
+        anno_info = {}
+        anno_info["info"] = {"description":"For testing COCO dataset only.",
+                             "year":2022,
+                             "date_created":"2022/03/24"}
+
+        anno_info["licenses"] = [{"url":"http://creativecommons.org/licenses/by-nc-sa/2.0/",
+                                  "id":1,
+                                  "name":"Attribution-NonCommercial-ShareAlike License"}]
+
+        anno_info["categories"] = [{"supercategory":"person",
```

Comment:
It seems that only COCO 17-kpt body pose definition is supported. The generality is a little bit concerning.